### DISCUSSION TITLE
Marlin Firmware Ender 3V2 / 3Neo / CR10-V3

### DISCUSSION THREAD

**Discussion by IridianZore** [mriscoc/Ender3V2S1#1143] [Q&A]
![image](https://github.com/mriscoc/Ender3V2S1/assets/145200529/02951d2b-d3a1-49a4-b0f0-cee13e99d050)

**Comment by IridianZore** [comment]
Hello friends,
Unfortunately, I can't get the firmware to work.
I have an:
 Ender 3V2 with a SKR mini E3 V3 board and a BTT TFT35 display. With BLTOUCH and without filament sensor.
Ender 3Neo with an SKR mini E3 V3 board and a BTT TFT35 display. With BLTOUCH and without filament sensor.
Ender CR-10 V3 with an SKR mini E3 V3 board and a BTT TFT35 display. With BLTOUCH and with filament sensor.
No matter which instructions I followed, whether the numerous videos from Youtube.
Or the Marlin Guide from Makenprint.
https://www.makenprint.uk/3d-printing/3d-firmware-guides/marlin/consolidated-marlin-guide/
It never worked. Either no bin file was created or the firmware was not running on the devices.
If I were to upload all the data for you, it would be gigabytes of attempts by now.
Perhaps one of you has your data and could make it available to me.
I'm really desperate as I've been working on it 24/7 for 3 weeks now but have hardly had any success.
I've tried everything I can think of, but I've only rooky and been 3D printing for 3 months.
FYI:
I have checked the wiring several times and it is correct. BLTOUCH is connected directly to the 5-pin connector.
The axes are connected correctly.
I have reversed the polarity of the extruder motor on the CR-10 V3 as it rotates inverted during feeding. 
Many thanks in advance for your support.

**Reply by mriscoc** [reply]
There isn't BTT TFT35 display support, check the requirements here: https://github.com/mriscoc/Ender3V2S1/wiki#requirements
